waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -784,9 +787,19 @@ proc mountRlnRelayDynamic*(node: WakuNode,
     rlnIndex = regIndexRes.value
     debug "peer is successfully registered into the membership contract"
   else:
+    info "Peer is already registered to the membership contract"
```

Comment:
Why not `debug`? also the same for other parts of the code?

Reply:
Was unaware of different logging levels. Referred to https://nim-lang.org/docs/logging.html#Level

Have made some changes accordingly. Please suggest if more needed.